chirp/drivers/ga510.py

```diff
@@ -245,7 +385,15 @@ def sync_out(self):
             LOG.exception('General failure')
             raise errors.RadioError('Failed to upload to radio: %s' % e)
 
+    # def process_mmap(self):
+        # self._memobj = bitwise.parse(MEM_FORMAT, self._mmap)
+        
     def process_mmap(self):
+        if self.MODEL == "GS-5B" or self.MODEL == "SHX-8800" or self.MODEL == "AC-580" or self.MODEL == "XTR-5":
```

Comment:
Please store the memory format on the class and just parse it here. So something like:
```
class RadiooddityGA510Radio(..):
    _mem_format = MODEL_GA510_FORMAT

    def process_mmap(self):
        self._memobj = bitwise.parse(BASE_FORMAT + self._mem_format)

class Senhaix8800Radio(RadioddityGA510Radio):
    _mem_format = MODEL_SHX8800_FORMAT
```